01_Data/src/layers/sequelize/sql/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Will drop all views skipping geography_columns and geometry_columns since they are required by postgis plugin
+ */
+export const dropAllViews = `
+DO $$
+DECLARE
+    r RECORD;
+BEGIN
+    FOR r IN (
+        SELECT table_schema, table_name 
+        FROM information_schema.views 
+        WHERE table_schema NOT IN ('pg_catalog', 'information_schema')
```

Comment:
🤣 "Does sequelize not have something to help here?" - Unfortunately not - type ORM does - https://orkhan.gitbook.io/typeorm/docs/view-entities and it also supports inheritance https://orkhan.gitbook.io/typeorm/docs/entity-inheritance and seeding and migrations (https://orkhan.gitbook.io/typeorm/docs/migrations) and everything is all part of one NPM package while unfortunately right now with `sequelize`, we also use `sequelize-typescript` and with the combination of the two - this is unfortunately all I can find / get to work. I did look to get views working using something coming from those two libs, but because of how they interact and the lack of existing support, this was the most straight forward way. Similar limitations apply to this other PR where we create the view - https://github.com/citrineos/citrineos-ocpi/pull/96. You can see in that PR how somewhat odd it looks where we override the table name: 
```
@Table({
  modelName: Views.ViewTransactionsWithPartyIdAndCountryCodes,
  createdAt: false,
  updatedAt: false
})
export class ViewTransactionsWithPartyIdAndCountryCode extends Model {
```
but luckily, when doing this, since it generates SQL, it still does `SELECT * FROM ${Views.ViewTransactionsWithPartyIdAndCountryCodes}` so it uses the view name as the table name and it works, but if that wasnt the case, then we wouldnt be able to create this Model class as easily

Reply:
PostgreSQL is already the only db that can support our current feature set, regardless of ORM we will be required to stick with Postgres or Postgres-like dbs.

Reply:
I am realizing here that we may have an issue here because if a COS user plugs COS into their existing PG DB then this block will drop their existing views if they had any. I guess it is better to adjust and use the `Views` enum to loop through the concrete views that we have in that enum and only drop those, I can make that adjustment 👍

Reply:
That's actually a really good point and a huge risk of interfering with other's view. 
It also feels weird to blanket drop a bunch of things. I found 2 alternative approaches:
1. Write a migration that drops the view in a migration file that then gets run
2. Add dropping the view to overriding the sync option of the Table annotated class. So when sync is set to true, it will also drop the views. 

You had also mentioned of prefixing views with `COS` or similar. But I personally am not a big fan of relying on that. It creates implicit knowledge about how outside components need to be designed as well...

Reply:
Yes option 2 is the right way to go most likely, either way though, I think I will close this PR since we do not need to account for views yet, and we can handle specifics if we do wish to implement any views or materialized views